src/streamlink/plugins/eltrecetv.py

```diff
@@ -0,0 +1,41 @@
+import re, html
+
+from streamlink.plugin import Plugin
+from streamlink.plugin.api import http, useragents, validate
+from streamlink.stream import HLSStream, HTTPStream
+from streamlink.utils import parse_json
+
+
+class ElTreceTV(Plugin):
+    _url_re = re.compile(r'https?://(?:www\.)?eltrecetv.com.ar/.+')
+
+    @classmethod
+    def can_handle_url(cls, url):
+        return cls._url_re.match(url)
+
+    def _get_streams(self):
+
+        video_url_found_hls = ""
+
+        if self.url[self.url.index('eltrecetv.com.ar/') +17:].startswith("vivo"):
+            http.headers = {'Referer': self.url,
+            'User-Agent': "ArtearPlayer/3.2.42 (Linux;Android 4.4.2) ExoPlayerLib/1.5.9"}
+            video_url_found_hls = "http://stream.eltrecetv.com.ar/live13/13tv/13tv1/playlist.m3u8"
+        else:
+            http.headers = {'Referer': self.url,
+            'User-Agent': useragents.CHROME}
+            res = http.get(self.url)
+            video_search = res.text
+            video_search = video_search[video_search.index('data-kaltura="') +14:]
+            video_search = video_search[: video_search.index('"')]
+            video_search = html.unescape(video_search)            
+            json_video_search = parse_json(video_search)
+            video_url_found_hls = "https://vodgc.com/p/111/sp/11100/playManifest/entryId/" + json_video_search["entryId"] + "/format/applehttp/protocol/https/a.m3u8"
+         
+        if video_url_found_hls:
+            hls_streams = HLSStream.parse_variant_playlist(self.session, video_url_found_hls)
+            for s in hls_streams.items():
+                yield s  
+               
```

Comment:
There is **whitespace**, it doesn't really affect the code.
But it looks better without it 👓 

![github1](https://cloud.githubusercontent.com/assets/24879726/25223093/b914c490-25ba-11e7-9cb1-d2e245e28a9b.png)